File: dotnet/MeetUp.Domain/Entities/Event.cs
namespace MeetUp.Domain.Entities
{
    public class Event : BaseEntity
    {
        public string Title { get; set; }
        public int CategoryId { get; set; }
        public int Capacity { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Image { get; set; }
        public DateTime Created { get; set; }
        public string Description { get; set; }
        public int UserId { get; set; }

    }
}

File: nodeJs/src/models/event.js
const mongoose = require('mongoose');

const eventSchema = new mongoose.Schema({
    title: {
        type: String,
        required: true
    },
    capacity: {
        type: Number,
        required: true
    },
    city: {
        type: String,
        required: true
    },
    state: {
        type: String,
        required: true
    },
    image: {
        type: String,
        required: true
    },
    description: {
        type: String,
        required: true
    },
    categories: {
        type: Object,
        required: true,
        ref: 'categories'
    },
    user: {
        type: Object,
        required: true,
        ref: 'user'
    },
    isDeleted: {
        type: Boolean,
        default: false
    },
    created: {
        type: String,
        default: false
    }
});

eventSchema.pre('find', function () {
    this.where({ isDeleted: false });
});

eventSchema.pre('findOne', function () {
    this.where({ isDeleted: false });
});

const event = mongoose.model('Event', eventSchema);

module.exports = event;